internal/controller/controllers/agent_controller.go

```diff
@@ -590,6 +621,37 @@ func requirementsMet(agent *aiv1beta1.Agent, status string) {
 	})
 }
 
+func bound(agent *aiv1beta1.Agent, status string, h *models.Host) {
+	var condStatus corev1.ConditionStatus
+	var reason string
+	var msg string
+	switch status {
+	case models.HostStatusBinding:
+		condStatus = corev1.ConditionFalse
+		reason = aiv1beta1.BindingReason
+		msg = aiv1beta1.BindingMsg
+	case models.HostStatusUnbinding:
+		condStatus = corev1.ConditionFalse
+		reason = aiv1beta1.UnbindingReason
+		msg = aiv1beta1.UnbindingMsg
+	case models.HostStatusDisconnectedUnbound, models.HostStatusKnownUnbound, models.HostStatusInsufficientUnbound,
+		models.HostStatusDisabledUnbound, models.HostStatusDiscoveringUnbound:
+		condStatus = corev1.ConditionTrue
```

Comment:
i think it is supposed to be ConditionFalse, no?

Reply:
Yep. Fixed and added tests.